docs/moduleAna1Tests.rst

```diff
@@ -3,172 +3,194 @@ ana1Tests: Testing
-analytical results and also to compare simulation results to the exact solutions.
-An example is given in runDamBreak.py, where the analytical solution is computed and
-avalanche simulations are performed and both results are then compared.
-The input data for this example can be found in ``data/avaDamBreak``.
+Two deviation/difference/error measures are used. The first one is based on the :math:`\mathcal{L}_{max}`
+norm (uniform norm), the second on the Euclidean norm (:math:`\mathcal{L}_{2}` norm). In both cases,
+the aim is to measure the deviation between a numerical solution and reference solution on an interval
+(one or two dimensional). Let :math:`f_{num}` be the numerical solution and
+:math:`f_{ref}` the reference solutions defined on an interval :math:`\Omega`.
+The deviation is defined by :math:`\epsilon(x) = f_{num}(x) - f_{ref}(x)`
 
-The :py:func:`ana1Tests.damBreak.plotResults` function generates profile plots for the flow depth and velocity and
-an animation of the flow depth evolution.
-With the :py:func:`ana1Tests.damBreak.plotComparison` function, the simulation results are plotted alongside the
-analytical solution for the given time step for both, flow depth and velocity.
-Details can be found here: :py:mod:`ana1Tests.damBreak`.
 
+Uniform norm
+--------------
 
-To run
-------
-An example on how to apply this test to :py:mod:`com1DFA` is provided in :py:mod:`runScripts/runDamBreak`.
-The required input files are located in ``data/avaDamBreak``, where the model configuration file can
-be found (``data/avaDamBreak/Inputs/damBreak_com1DFACfg.ini``). In this configuration file, there
-is a specific section ``'DAMBREAK'`` providing the required input parameters to compute the analytical solution.
-In order to run the test example:
-
-* in ``AvaFrame/avaframe`` run::
+The :math:`\mathcal{L}_{max}` norm measures the largest absolute value:
 
-    python3 runScripts/runDamBreak.py
+.. math::
+    \mathcal{L}_{max}(\epsilon(x)) = \sup_{x\in \Omega}(\vert \epsilon(x) \vert)
 
-Analytical solution of dam break problem, flow depth and flow velocity at 0.5 s.
+This will give an idea of the largest difference between the two solutions.
+It can be applied to one or two dimensional results.
 
-.. list-table::
-    :widths: 50 50
+It can also be normalized by dividing the uniform norm of the deviation
+by the uniform norm of the reference:
 
+.. math::
+    \frac{\mathcal{L}_{max}(\epsilon(x))}{\mathcal{L}_{max}(f_{ref}(x))}
 
-    * -
 
-        .. figure:: _static/damBreakFlowDepth.png
-            :width: 100%
+Euclidean norm
+---------------
 
-            Flow depth in x direction after 5s
+The :math:`\mathcal{L}_{2}` norm is defined by:
 
+.. math::
+    \mathcal{L}_{2}(\epsilon) = \int_{x\in \Omega}\vert\vert \epsilon(x) \vert\vert^2\,dx
 
-      -
+This norm will give an overall measure of the deviations. It is useful to
+normalize the norm either by dividing the norm of the deviation by the norm of
+the reference solution:
```

Comment:
It is useful to normalize the norm of the deviation either by dividing with the norm of the reference solution: ... like this?